<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(&quot;light_background.jpg&quot;)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>451</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Aspekt&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_word">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>480</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Wortschatz&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="indent">
       <number>-11</number>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_expain">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>480</width>
        <height>90</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Explaination&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_verben">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>491</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>Infinitiv</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_verben_title">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>141</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>    Infinitiv</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_verben_prs">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>481</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>Verben</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_verben_title_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>140</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>Präteritum</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_verben_title_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>101</width>
        <height>80</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>Präsens</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_verben_title_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>Perfekt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_verben_prm">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>160</y>
        <width>531</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>Verben</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_verben_pft">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>501</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url()</string>
      </property>
      <property name="text">
       <string>Verben</string>
      </property>
      <property name="openExternalLinks">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_verben_next">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url()</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-QT_Test1-Desktop_Qt_5_3_MinGW_32bit-Debug/Button-Next-icon.png</normaloff>../build-QT_Test1-Desktop_Qt_5_3_MinGW_32bit-Debug/Button-Next-icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_verben_last">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>490</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url()</string>
    </property>
    <property name="text">
     <string>Last</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-QT_Test1-Desktop_Qt_5_3_MinGW_32bit-Debug/Button-Previous-icon.png</normaloff>../build-QT_Test1-Desktop_Qt_5_3_MinGW_32bit-Debug/Button-Previous-icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_hide">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url()</string>
    </property>
    <property name="text">
     <string>Hide Explaination</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_online">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url()</string>
    </property>
    <property name="text">
     <string>Online Explaination</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../build-QT_Test1-Desktop_Qt_5_3_MinGW_32bit-Debug/Search-icon.png</normaloff>../build-QT_Test1-Desktop_Qt_5_3_MinGW_32bit-Debug/Search-icon.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_I_remember">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url()</string>
    </property>
    <property name="text">
     <string>I remember!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url()</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Gerneral</string>
    </property>
    <addaction name="actionStart_Word_Excersice"/>
    <addaction name="actionStart_Verben_Excersice"/>
    <addaction name="actionBack_to_main_menu"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>Configure</string>
    </property>
    <widget class="QMenu" name="menuKapitel">
     <property name="title">
      <string>Kapitel</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionKapitel_1"/>
     <addaction name="actionKapitel_2"/>
     <addaction name="actionKapitel_3"/>
     <addaction name="actionKapitel_4"/>
     <addaction name="actionKapitel_5"/>
     <addaction name="actionKapitel_6"/>
     <addaction name="actionKapitel_7"/>
     <addaction name="actionKapitel_8"/>
     <addaction name="actionKapitel_9"/>
     <addaction name="actionKapitel_10"/>
    </widget>
    <widget class="QMenu" name="menuSprach">
     <property name="title">
      <string>Sprach</string>
     </property>
     <addaction name="actionChinesisch"/>
     <addaction name="actionEnglisch"/>
     <addaction name="actionDeutsch"/>
    </widget>
    <addaction name="actionLehrbuch"/>
    <addaction name="menuKapitel"/>
    <addaction name="menuSprach"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuConfigure"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart_Word_Excersice">
   <property name="text">
    <string>Start Word Excersice</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionBack_to_main_menu">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Back to main menu</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionStart_Verben_Excersice">
   <property name="text">
    <string>Start Verben Excersice</string>
   </property>
  </action>
  <action name="actionLehrbuch">
   <property name="text">
    <string>Lehrbuch</string>
   </property>
  </action>
  <action name="actionKapitel_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 1</string>
   </property>
  </action>
  <action name="actionKapitel_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 2</string>
   </property>
  </action>
  <action name="actionKapitel_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 3</string>
   </property>
  </action>
  <action name="actionKapitel_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 4</string>
   </property>
  </action>
  <action name="actionKapitel_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 5</string>
   </property>
  </action>
  <action name="actionKapitel_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 6</string>
   </property>
  </action>
  <action name="actionKapitel_7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 7</string>
   </property>
  </action>
  <action name="actionKapitel_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 8</string>
   </property>
  </action>
  <action name="actionKapitel_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 9</string>
   </property>
  </action>
  <action name="actionKapitel_10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Kapitel 10</string>
   </property>
  </action>
  <action name="actionChinesisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinesisch</string>
   </property>
  </action>
  <action name="actionEnglisch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Englisch</string>
   </property>
  </action>
  <action name="actionDeutsch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Deutsch</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
